.travis.yml

```diff
@@ -9,34 +9,55 @@ cache:
 env:
   global:
     - SYMFONY_DEPRECATIONS_HELPER="weak_vendors"
+    - ENABLE_CODE_COVERAGE="false"
 
 matrix:
   fast_finish: true
   include:
     - php: 7.1
-      env: SYMFONY_VERSION=4.0.*
     - php: 7.1
+      env:
+        - SYMFONY_VERSION=4.0.*
+        - COMPOSER_FLAGS="--prefer-lowest"
+        - DEPENDENCIES="symfony/phpunit-bridge:^4"
+    - php: 7.2
+    - php: 7.2
       env: SYMFONY_VERSION=3.4.*
     - php: 7.2
       env: SYMFONY_VERSION=4.0.*
     - php: 7.2
-      env: SYMFONY_VERSION=3.4.*
+      env:
+        - DEPENDENCIES="symfony/phpunit-bridge:^4"
+        - COMPOSER_UPDATE_OPTIONS="--no-dev"
     - php: 7.2
-      env: SYMFONY_VERSION=dev-master
+      env:
+        - ENABLE_CODE_COVERAGE="true"
+    - php: 7.2
+      env:
+        - SYMFONY_VERSION=dev-master
+        - STABILITY=dev
     - php: nightly
-      env: SYMFONY_VERSION=4.0.*
-  allow_failures: 
-    - env: SYMFONY_VERSION=dev-master
+  allow_failures:
+    - env:
+      - ENABLE_CODE_COVERAGE="true"
+    - env:
+        - SYMFONY_VERSION=dev-master
+        - STABILITY=dev
     - php: nightly
-    
+
 before_install:
-  - bash ./.travis/exec-before.bash
+  - if [[ "$SYMFONY_VERSION" != "" ]]; then travis_retry composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update $COMPOSER_FLAGS; fi;
+  - if [[ "$DEPENDENCIES" != "" ]]; then travis_retry composer require ${DEPENDENCIES} --no-update $COMPOSER_FLAGS; fi;
+  - if [[ "$STABILITY" != "" ]]; then composer config minimum-stability $STABILITY; fi
+  - if [[ "$ENABLE_CODE_COVERAGE" != "true" && "$TRAVIS_EVENT_TYPE" != "cron" ]]; then rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini; fi;
```

Comment:
Instead of `rm -f ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/xdebug.ini` you should use the `phpenv` configuration plugin instead; see ["Preinstalled PHP extensions" (blog.travis-ci.com)](https://blog.travis-ci.com/2013-03-08-preinstalled-php-extensions).

```
phpenv config-rm xdebug.ini
```